src/EluvioLive.js

```diff
@@ -593,10 +630,9 @@ class EluvioLive {
 	  } catch(e) {
 	  }
 
-	  console.log(i, tokenId.toString(), "hold: ", holdSecs.toString(), holdEnd);
-
+	  balance[tokenId.toString()] = {hold: holdSecs.toString(), holdEnd: holdEnd};
```

Comment:
We're keying by tokenId? Maybe key by nft address and add the tokenId into the value object.

Reply:
Ok - I changed it to an array and the token ID is in the object

Example:

```
./elv-live nft_balance_of 0x98ed26d3ba8ed031de4e347f8428def7fce21efa 0x6FcfFb1d00648Ba6777e8e0398410B973665E2e2
NFT - call 0x98ed26d3ba8ed031de4e347f8428def7fce21efa 0x6FcfFb1d00648Ba6777e8e0398410B973665E2e2
Network: demov3
- tokenId: '776'
  hold: '1643934785'
  holdEnd: 2022-02-04T00:33:05.000Z
- tokenId: '1000776'
  hold: '1643934785'
  holdEnd: 2022-02-04T00:33:05.000Z
- tokenId: '2000776'
  hold: '1643934785'
  holdEnd: 2022-02-04T00:33:05.000Z
- tokenId: '777'
  hold: '1643934964'
  holdEnd: 2022-02-04T00:36:04.000Z
- tokenId: '1000777'
  hold: '1643934964'
  holdEnd: 2022-02-04T00:36:04.000Z
- tokenId: '2000777'
  hold: '1643934964'
  holdEnd: 2022-02-04T00:36:04.000Z
- tokenId: '778'
  hold: '1643935145'
  holdEnd: 2022-02-04T00:39:05.000Z
- tokenId: '1000778'
  hold: '1643935145'
  holdEnd: 2022-02-04T00:39:05.000Z
- tokenId: '2000778'
  hold: '1643935145'
  holdEnd: 2022-02-04T00:39:05.000Z
- tokenId: '732'
  hold: '1643835003'
  holdEnd: 2022-02-02T20:50:03.000Z
- tokenId: '780'
  hold: '1643952305'
  holdEnd: 2022-02-04T05:25:05.000Z
```